autoload/maktaba/extension.vim

```diff
@@ -0,0 +1,159 @@
+""
+" @dict ExtensionRegistry
+" A registry for the extensions used a single plugin.  Extensions are
+" dictionaries with a plugin-specific interpretation
+"
+" Plugins should use |Plugin.GetExtensionRegistry| to gain access to their own
```

Comment:
That didn't, but `@function(Plugin.GetExtensionRegistry)` did (and translates to the right tag name,  too, unlike what I had before).  Thanks!